mobile/native/migrations/2024-04-16-144000_add_accumulated_order_matching_fee_to_position/up.sql

```diff
@@ -0,0 +1,11 @@
+-- Your SQL goes here
+ALTER TABLE positions
+    ADD COLUMN order_matching_fees BIGINT NOT NULL DEFAULT 0;
+
+-- To avoid division by zero.
+UPDATE positions
+SET order_matching_fees = 0 where average_entry_price=0;
+
+-- Before this migration, the taker fee was hard-coded to 0.30%.
+UPDATE positions
+SET order_matching_fees = quantity * (1 / average_entry_price) * 0.0030 where average_entry_price!=0;
```

Comment:
Yeah, when I've written it 2.2.0 wasn't released yet. 🙈

Reply:
Also, any trade between Friday, 12.04 and now has no order matching fee

Reply:
I don't think it's too critical as it will only move the max quantity lower. Do the user might not be able to trade everything, but he won't run into errors.